keps/sig-scheduling/20200612-simplified-topology-manager.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Simplified version of TopologyManager in kube-scheduler
+authors:
+  - "@AlexeyPerevalov"
+  - "@swatisehgal"
+owning-sig: sig-scheduling
+participating-sigs:
+reviewers:
+  - "@ahg-g"
+  - "@huang-wei"
+  - "@derekwaynecarr"
+approvers:
```

Comment:
i think this needs an approver from sig-node that maintains topology manager node local scheduling.

i recommend @klueska

Reply:
Will do!